sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -135,38 +135,5 @@
         </dependency>
       </dependencies>
     </profile>
-
-    <profile>
-      <id>java9plus</id>
-      <activation>
-        <jdk>[9,)</jdk>
-      </activation>
```

Comment:
Just curious, why did we have this before for appconfig?

Reply:
This resolved issue in test-compile with regards to Java 9+'s module system. With the change to disable the module system in test code this is no longer an issue.